pkg/controllers/sync/accessor.go

```diff
@@ -154,23 +159,22 @@ func (a *resourceAccessor) Run(stopChan <-chan struct{}) {
 func (a *resourceAccessor) HasSynced() bool {
 	kind := a.typeConfig.GetFederatedType().Kind
 	if !a.versionManager.HasSynced() {
-		klog.V(2).Infof("Version manager for %s not synced", kind)
+		a.logger.WithValues("federated-type-kind", kind).V(3).Info("Version manager for federated-type-kind not synced")
```

Comment:
`"federated-type-kind"` is unnecessary since the logger should contain ftc name.